### PR TITLE
Print sonobuoy data

### PR DESCRIPTION
**Pull request by TerryHowe**
Print the various sonobuoy calls to make it clear what is going on.

### CODE REVIEW

File: pkg/executables/sonobuoy.go [hunk 1]
```diff
@@ -31,19 +31,23 @@ func (k *Sonobuoy) Run(ctx context.Context, contextName string, args ...string)
...
 	}
 	executionArgs = append(executionArgs, args...)
-	_, err := k.executable.Execute(ctx, executionArgs...)
+	output, err := k.executable.Execute(ctx, executionArgs...)
+	command := strings.Join(executionArgs, " ") + "\n"
 	if err != nil {
-		return "", fmt.Errorf("error executing sonobuoy: %v", err)
+		return command, fmt.Errorf("error executing sonobuoy: %v", err)
 	}
+	return command + output.String(), err
```
Comment (abhinavmpandey08) [inline pkg/executables/sonobuoy.go:39]: should we add a delimiter between command and output.String(), maybe a new line?

File: pkg/executables/sonobuoy.go [hunk 2]
```diff
@@ -55,8 +59,9 @@ func (k *Sonobuoy) Run(ctx context.Context, contextName string, args ...string)
 		outputFile,
 	}
 	output, err = k.executable.Execute(ctx, executionArgs...)
+	command := strings.Join(executionArgs, " ") + "\n"
 	if err != nil {
-		return "", fmt.Errorf("error executing sonobuoy results command: %v", err)
+		return command, fmt.Errorf("error executing sonobuoy results command: %v", err)
 	}
-	return output.String(), err
+	return command + output.String(), err
```
Comment (abhinavmpandey08) [inline pkg/executables/sonobuoy.go:66]: same as above

File: pkg/executables/sonobuoy.go (hunk 1 above)
Comment (TerryHowe) [reply pkg/executables/sonobuoy.go:39]: Above `command := strings.Join(executionArgs, " ") + "\n"`

Comment (abhinavmpandey08) [reply pkg/executables/sonobuoy.go:39]: oh I see nvm then

**Review by abhinavmpandey08 — APPROVED** [review]

**Pull request merged**